Temmietayo/All-Recipes
# Ch user profile template 169997289

#### What does this PR do?
Add the template for user to view their profilr .

#### Description of Task to be completed?
Authenticated User should be able to view his/her profile

#### How should this be manually tested?

- clone the repo

- cd into the template directory

- open the userprofile.html with any browser of your choice

- these template should open in that browser
#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
[169997289](https://www.pivotaltracker.com/story/show/169997289)
